### ISSUE TITLE
Cyberflix Catalog

### ISSUE THREAD

**Issue by marcojoao** [danamag/stremio-addons-list#119]
### Addon Manifest URL

https://82d7ae415a21-llama-catalog.baby-beamup.club/manifest.json

### Addon Description

Discover a vast selection of Movies and TV shows from popular streaming services like Netflix, Disney Plus, HBO Max, Amazon Prime, and more with Cyberflix.

### Language of Content

English (en-GB)

### Choose Labels

- [X] anime
- [X] asian drama
- [ ] bollywood
- [X] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [X] metadata
- [X] misc
- [X] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [X] tv shows
- [ ] video games

**Comment by marcojoao** [comment]
# Preview
![alt text](https://i.postimg.cc/65Bz75QQ/Screenshot-2023-04-10-at-12-48-59.png "Cyberflix Catalog")

**Comment by Zero0Q** [comment]
Would you be able to add an option in configuration window for us to be able to chose which catalog we would like to show?

**Comment by Flyologist** [comment]
Thank you very much...I searched everywhere for this..was working and stopped

**Comment by marcojoao** [comment]
> Thank you very much...I searched everywhere for this..was working and stopped

Hello I did a full restart, it should be working right now

**Comment by marcojoao** [comment]
> Would you be able to add an option in configuration window for us to be able to chose which catalog we would like to show from popular and new.

Hello thank you for the feedback, that feacture is already on my backlog, will add that option when I get some time.

**Comment by Zero0Q** [comment]
Hi, I was wondering where can i fork this source code? Thanks

**Comment by shanicenaomihaye** [comment]
Having an issue that's started today where I'm getting 'empty content' under most of the categories. Is this just on my end or a global issue?

**Comment by TheDon07** [comment]
First off, thanks for making one of the best addons for Stremio. It looks like the addon was updated recently as I see new options when trying to configure. When I try to change any of the default options, I keep getting a ERR_OPENING_MEDIA in Stremio and the webpage gives me a bad gateway. Is there a way to get an older version in the meantime? I appreciate your help.

**Comment by marcojoao** [comment]
Hello, thank you for your kind words.

The problem is related with the beamup service, not the addon itself, let me check what is happening, thank you for the recall.

**Comment by TheDon07** [comment]
> Hello, thank you for your kind words.
> 
> The problem is related with the beamup service, not the addon itself, let me check what is happening, thank you for the recall.

Thanks for the quick reply. I was setting up Stremio for a friend and this is necessary addon. I hope it is an easy fix!

**Comment by TheDon07** [comment]
@marcojoao I just tried it again and it's working. So whatever you did, did the trick! You're the best!

**Comment by EroSDEV** [comment]
Awesome job! thank you very much. Is there any possibility to get the titles or descriptions in other languages like Spanish?

**Comment by Mahmoud23go** [comment]
G

**Comment by qwerty2k** [comment]
If I may ask where does the catalogue get its data from? Tmdb? Trakt?

**Comment by ericvlog** [comment]
hello, great addons , it is posible to only have option install meta of the anime, ....because the anime catalog so much for me, but i like the discover tab anime of cyberflix.

**Comment by fruitvenison** [comment]
Is it possible to remove the 'new' section for each service i.e New Movies - Amazon Prime & New Movies - Apple TV Plus
I have unticked 'Premieres' when setting up the add-on via Stremio but these sections still appear.
Thanks!

**Comment by marcojoao** [comment]
> Is it possible to remove the 'new' section for each service i.e New Movies - Amazon Prime & New Movies - Apple TV Plus I have unticked 'Premieres' when setting up the add-on via Stremio but these sections still appear. Thanks!

I add more control to the config page, please configure again

**Comment by fruitvenison** [comment]
> > Is it possible to remove the 'new' section for each service i.e New Movies - Amazon Prime & New Movies - Apple TV Plus I have unticked 'Premieres' when setting up the add-on via Stremio but these sections still appear. Thanks!
> 
> I add more control to the config page, please configure again

Thank you this is excellent!

**Comment by Zero0Q** [comment]
This is perfect now, Thank you so Much😎

**Comment by MrSick0ff** [comment]
Doesn’t seem to change anything for me

**Comment by MrSick0ff** [comment]
I was mistaking

**Comment by a95tvbox** [comment]
![issue](https://github.com/danamag/stremio-addons-list/assets/158166844/ab3dbd73-5e0a-4a9b-968d-431425ca131e)
@marcojoao Hey bro I'm going through this issue now when i'm trying to install new version of Cyberflix Catalog v1.5.0 ... Please help me with this...

**Comment by Aziefwa** [comment]
> ![issue](https://private-user-images.githubusercontent.com/158166844/303864009-ab3dbd73-5e0a-4a9b-968d-431425ca131e.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc2MDE5MzcsIm5iZiI6MTcwNzYwMTYzNywicGF0aCI6Ii8xNTgxNjY4NDQvMzAzODY0MDA5LWFiM2RiZDczLTVlMGEtNGE5Yi05NjhkLTQzMTQyNWNhMTMxZS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMFQyMTQ3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NDMxMzgzOGVkZWI1MTFkYTk3OTljYTY1ZWUwMTFhNDU4MTIwMjNhODkyOTc3ZTU5ZmEzNjdhOWVlY2EzMzlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ODMvgxebTcFK9N9s8tpunGoCXVdsN0tgyJQLuQ80Azc) @marcojoao Hey bro I'm going through this issue now when i'm trying to install new version of Cyberflix Catalog v1.5.0 ... Please help me with this...



> Can't seem to make it work, installed - uninstalled a few times and nothing shows up. And no option to configure.
> 
> The "configure" glitches and won't fully open.

Same here

**Comment by Georgescu0001** [comment]
The Add-on Ver.1.5.0 is not functioning properly, on PC once installed it works while you're actively browsing categories however closing the Stremio program deletes the add-on from the menu as if it was uninstalled by the user. On the mobile app it straight doesn't work at all. 
Unfortunately even previous versions seemed to work on and off, the add-on is unreliable. A more permanent fix would be greatly appreciated if possible thanks!

**Comment by a95tvbox** [comment]
Please fix this Cyberflix Catalog v1.5.0. This is one of the best catalog we all needed...

**Comment by jay78887** [comment]
Having issues all week with custom config.
Getting ADDON PUSH FAILED error.
Constantly uninstalling, reinstalling. Disappears after logging out.

**Comment by marcojoao** [comment]
> Having issues all week with custom config. Getting ADDON PUSH FAILED error. Constantly uninstalling, reinstalling. Disappears after logging out.

Throughout this week, I've been grappling with a recurring issue tied to a custom configuration. Specifically, I keep encountering the "ADDON PUSH FAILED" error, which has led to a frustrating cycle of uninstalling and reinstalling. Strangely, the problem seems to vanish upon logging out.

Regarding the error, it stems from a limitation within Stremio itself, primarily due to the size of the catalog. Essentially, while the addon installation process succeeds initially, the client faces a roadblock when attempting to sync with the server responsible for managing all addons. This bottleneck occurs because the configuration file is too extensive.

Currently, the workaround involves dividing your configuration into smaller segments. For instance, you could install one set of catalogs on one instance and another set on a separate instance. In simpler terms, the solution entails installing the addon multiple times. While admittedly not perfect, it's the best approach available to us at the moment.

**Comment by martinboson** [comment]
addon not working.

"oops! addon returned empty content"
I even reinstalled it with the trakt code. nothing.
win10. using stremio + torrentio + realdebrid

**Comment by lothar-cell** [comment]
Favourite add-on:) Is the new 40 catalogue-cap permanent?
What are the sources of the Recommendations and the Trending catalogue?

**Comment by Flyologist** [comment]
Hi brother,

just a suggestion a lot of shows are not playing and can you change the icon selection to a brighter colour as you go from tv shows and movies etc this way you can see what to click ?


Sent with Spark

**Comment by morrogenduath** [comment]
![image](https://github.com/danamag/stremio-addons-list/assets/46622026/51d735b9-c9d4-49eb-a1b8-9242eafee0e4)
Version in the manifest.json = "Unkown" which doesn't let Stremio install this addons.

**Comment by coolspot18** [comment]
Setup screen is blank - can't select catalogs.
![image](https://github.com/danamag/stremio-addons-list/assets/5160442/acd151ad-a6e7-4a86-a90c-eec6c585ec35)

**Comment by danielRep** [comment]
> Setup screen is blank - can't select catalogs. 

I'm experiencing the same issue.

**Comment by Kelpe1925** [comment]
1.5.1 no channels to add.

**Comment by Hal0414** [comment]
Same thing for me as well

**Comment by javiimp** [comment]
is there a way to show info in spanish? Or a future update, many thanks in advance

**Comment by kkerasounlis** [comment]
Jeopardy episodes don't show up anymore after the change in seasons that happened today

**Comment by captaintourettes** [comment]
Request to add more services like Cinemax, and The Movie Channel. Also possibly more categories under some of the services. For example it's hard to find programs under the channels in Discovery+ Search his hit or miss sometimes. THANKS!

**Comment by wb4r** [comment]
This addon is unusable atm. Not working in app and config page is either blank or 500 crash

**Comment by mindygame** [comment]
It seems that the addon is down, not able to load the catalog list to pick for a new installation.

**Comment by XDeadHackerX** [comment]
Hi marcojoao, thanks for this fantastic Addons and my congratulations for your well worked preconfiguration prior to installation, its good compatibility and filtering with Stremio. 

According to javiimp it would be a very remarkable point the translation to Spanish, our community uses your Addon but we would like to have the possibility to read the synopsis in our language. I don't know very well how it could be done but what I'm sure is that TMDB has such translations and filtering by platform, so it could be an option.

Thank you very much for your work.

**Comment by tomtomclub** [comment]
https://discord.com/channels/667359689780101130/1282631763549487175/1282631763549487175.    Its works Yes, without this addon?

**Comment by Djlilyazii** [comment]
Is this add on still working? Its not updating for some if us.

**Comment by MrSick0ff** [comment]
Mine doesn't work

**Comment by jfurlan55** [comment]
This addon do not updated the library. Is abandoned

**Comment by Sophanos** [comment]
This add-on does not work, skip.

**Comment by lothar-cell** [comment]
It does for me. There's also an Elfhosted link somewhere, maybe try that?

**Comment by Sophanos** [comment]
> It does for me. There's also an Elfhosted link somewhere, maybe try that?

It works but does not update. The Koyeb version of Cyberflix works and updates but is slow while elfhosted works but does not update.

**Comment by crazyrabbit0** [comment]
### The beta server works fine, you can get it from: https://catalog-cyberflix.koyeb.app/

The elfhosted server is stuck for some reason...

As stated by Cyberflix's developer:
https://www.reddit.com/r/StremioAddons/comments/1caxgkp/cyberflix_updating/

**Comment by crazyrabbit0** [comment]
> > ### The beta server works fine, you can get it from: https://catalog-cyberflix.koyeb.app/
> > The elfhosted server is stuck for some reason...
> > As stated by Cyberflix's developer: https://www.reddit.com/r/StremioAddons/comments/1caxgkp/cyberflix_updating/
> 
> The Elfhost team replied in Discord that they have tried to contact the Cyberflix developer, but the developer is not responding to resolve this issue.

Oh really? Interesting, good to know 👌

I thought the developer was trying to fix this, at least that's what I understood from his last response:
https://www.reddit.com/r/StremioAddons/comments/1f8zk59/comment/lllswzl/

**Comment by Sophanos** [comment]
It works again, the developer has not given up on the project!

**Comment by lebigboss59** [comment]
Thank you so much

![image](https://github.com/user-attachments/assets/3dad97ad-e4f2-43a9-a311-d930c4dd05e3)


![image](https://github.com/user-attachments/assets/f866e7f5-b4bf-4685-8b9d-24fd7b5bd107)

**Comment by lebigboss59** [comment]
any chance to have this addon in french one day ?

**Comment by lebigboss59** [comment]
Hi, can we have a category with everything new or popular from Apple+, Netflix, Paramount+, ... and more ?
